tls/src/main/java/org/bouncycastle/tls/SessionID.java

```diff
@@ -4,7 +4,7 @@
 import org.bouncycastle.util.encoders.Hex;
 
 public final class SessionID
-    implements Comparable
+    implements Comparable<SessionID>
```

Comment:
TLS currently still has to compile under Java 1.4 i.e. without generics. Please remove this file from the PR.

Reply:
Done.